.eslintrc.json

```diff
@@ -47,10 +49,6 @@
     "@typescript-eslint/no-use-before-define": "off",
     "@typescript-eslint/no-empty-function": "off",
     "@typescript-eslint/no-explicit-any": "off",
-    "jest/consistent-test-it": [
-      "error",
-      { "fn": "it", "withinDescribe": "it" }
-    ],
```

Comment:
Can we keep this? (Unless there's an alternative without jest plugin.)

Reply:
I don't think it'd work with vitest, does it?

Reply:
I think it does.

Reply:
Did it with eslint-plugin-vitest

Reply:
🎉